2017/AUG/LTIME51/MATPAN/MATPAN.cpp

```diff
@@ -0,0 +1,27 @@
+//Made by vanguard7
+#include <bits/stdc++.h>
+using namespace std;
+
+int main() {
+	int t;
+	cin >> t;
+	while(t--) {
+		long long int prices[26], sum = 0;
```

Comment:
`int` is sufficient

Reply:
I guess it would change the running time of the code, which isn't required here. 😄